vulnerability/vulnerability_types.go

```diff
@@ -0,0 +1,10 @@
+package vulnerability
+
+import (
+	"github.com/elastic/cloudbeat/resources/providers/awslib/ec2"
+)
+
+type VulnSnapWithIns struct {
+	ins  ec2.Ec2Instance
+	snap ec2.EBSSnapshot
```

Comment:
`ec2.EBSSnapshot` was created exactly for that purpose, instead of having this new type `VulnSnapWithIns` we can just have more fields on `ec2.EBSSnapshot` that specify the Ec2Name and Ec2Id